### PR TITLE
Add service-apis image pushing job

### PR DESCRIPTION
**Pull request by aojea**
xref https://github.com/kubernetes-sigs/service-apis/issues/3

### CODE REVIEW

**Comment by BenTheElder** [comment]
cc @aledbf - IIRC you were also working with this right?

**Comment by aledbf** [comment]
@BenTheElder no, I was working in this https://github.com/kubernetes/test-infra/pull/15699

File: config/jobs/image-pushing/k8s-staging-service-apis.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+postsubmits:
+  # this is the github repo we'll build from; this block needs to be repeated for each repo.
+  kubernetes-sigs/k8s-container-image-promoter:
+    - name: service-apis-postsubmit-push-to-staging
+      cluster: test-infra-trusted
```
Comment (fejta) [inline config/jobs/image-pushing/k8s-staging-service-apis.yaml]: Jobs that run in the trusted cluster need to be defined in the trusted file

Comment (aojea) [reply config/jobs/image-pushing/k8s-staging-service-apis.yaml]: I've followed the instructions from https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md

How do I add the job to the "trusted" file :sweat_smile: ?

**Pull request merged**